lib/r10k/git/cache.rb

```diff
@@ -16,7 +16,7 @@ class R10K::Git::Cache
 
   include R10K::Settings::Mixin
 
-  def_setting_attr :cache_root, File.expand_path(ENV['HOME'] ? '~/.r10k/git': '/root/.r10k/git')
+  def_setting_attr :cache_root, File.expand_path(ENV['HOME'] ? '~/.r10k/cache': '/root/.r10k/cache')
```

Comment:
Ah. forge tarball caching happens in a directory named for the module, ex branan-eight_hundred-8.0.0/, and git module and control caching happens in a directory named for the repo, ex. git@github.com-andersonmills-r10k-testing.git/.